addons/account/models/chart_template.py

```diff
@@ -179,7 +179,7 @@ def _load(self, template_code, company, install_demo):
         template_data = data.pop('template_data')
 
         if reload_template:
-            self._pre_reload_data(company, template_data, data)
+            data = self._pre_reload_data(company, template_data, data)
```

Comment:
```suggestion
            self._pre_reload_data(company, template_data, data)
```
No need to assign data.